### ISSUE TITLE
Cannot find module net

### ISSUE THREAD

**Issue by Quenos** [jaggedsoft/node-binance-api#481]
I'm creating an app with electron and node.js

Installed node-binance-api

Compiled perfectly

Calling the API produces the following error in chrome:

Uncaught (in promise) Error: Cannot find module 'net'
    at webpackEmptyContext

**Comment by jaggedsoft** [comment]
I managed to get this working using electron in the past, but that was some time ago

**Comment by Quenos** [comment]
That's cool, but how? :)

**Comment by alonep56** [comment]
hello, I have the same error as you. Did you find the solution?

**Comment by Quenos** [comment]
@alonep56 No, but I gave up on using libraries for exchange APIs. Coding it yourself is almost as fast as troubleshooting the existing libs IMO and you can make it exactly as you like

**Comment by jaggedsoft** [comment]
I think this is the route I took as well. Since the original is designed for node and electron needs something designed for the browser
